### PR TITLE
Add an example of simple configuration inheritance

### PR DESCRIPTION
**Pull request by ctompkinson**

### CODE REVIEW

File: examples/pipeline-templates-v1/configuration-inheritance/template.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  triggers:
+    - type: jenkins
+      job: "{{ jenkinsTriggerJob }}"
+      master: "{{ jenkinsMaster }}"
+      propertyFile: "{{ jenkinsPropertyFile }}"
+      enabled: true
+  parameters:
+    - name: baseOs
+      description: The baseOs to bake the image with
+  conurrentExecutions:
```
Comment (emjburns) [inline examples/pipeline-templates-v1/configuration-inheritance/template.yml]: Should be "concurrentExecutions" (typo)

File: examples/pipeline-templates-v1/configuration-inheritance/configuration-inheritance-config.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+schema: "1"
+pipeline:
+  application: orca
+  name: Simple Configuraiton Inheritance Example
+  template:
+    source: file://template.yml
+  variables:
+    jenkinsTriggerJob: 'JobName'
+    jenkinsMaster: 'JenkinsMasterName'
+    propertyFile: 'properties.json'
```
Comment (emjburns) [inline examples/pipeline-templates-v1/configuration-inheritance/configuration-inheritance-config.yml]: should be "jenkinsPropertyFile"

**Review by emjburns — APPROVED** [review]
Thanks for this! Just two changes to make it valid and then LGTM

File: examples/pipeline-templates-v1/configuration-inheritance/template.yml (hunk 1 above)
Comment (ctompkinson) [reply examples/pipeline-templates-v1/configuration-inheritance/template.yml]: Ahha thanks for spotting those! That would of been tomorrows bug in my template :)

File: examples/pipeline-templates-v1/configuration-inheritance/template.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - type: jenkins
+      job: "{{ jenkinsTriggerJob }}"
+      master: "{{ jenkinsMaster }}"
+      propertyFile: "{{ jenkinsPropertyFile }}"
+      enabled: true
+  parameters:
+    - name: baseOs
+      description: The baseOs to bake the image with
+  concurrentExecutions:
+    parraellel: true
```
Comment (robzienert) [inline examples/pipeline-templates-v1/configuration-inheritance/template.yml]: Typo here - but good news, you don't need `parallel` at all anymore - we removed this flag from Spinnaker entirely.

**Comment by ctompkinson** [comment]
@robzienert Okay I've removed the parallel flag 👍

**Pull request merged**